a:5:{i:0;s:154:"Error in file: "C:\xampp\htdocs\Cybercom\php\magento\app\code\local\Cybercom\Salesman\sql\salesman_setup\mysql4-upgrade-1.0.2-1.0.3.php" - Wrong entity ID";i:1;s:1058:"#0 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\Resource\Setup.php(644): Mage::exception('Mage_Core', 'Error in file: ...')
#1 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\Resource\Setup.php(437): Mage_Core_Model_Resource_Setup->_modifyResourceDb('upgrade', '1.0.2', '1.0.3')
#2 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\Resource\Setup.php(320): Mage_Core_Model_Resource_Setup->_upgradeResourceDb('1.0.2', '1.0.3')
#3 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\Resource\Setup.php(235): Mage_Core_Model_Resource_Setup->applyUpdates()
#4 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\App.php(444): Mage_Core_Model_Resource_Setup::applyAllUpdates()
#5 C:\xampp\htdocs\Cybercom\php\magento\app\code\core\Mage\Core\Model\App.php(370): Mage_Core_Model_App->_initModules()
#6 C:\xampp\htdocs\Cybercom\php\magento\app\Mage.php(686): Mage_Core_Model_App->run(Array)
#7 C:\xampp\htdocs\Cybercom\php\magento\index.php(83): Mage::run('', 'store')
#8 {main}";s:3:"url";s:104:"/Cybercom/php/magento/index.php/admin_salesman/adminhtml_index/new/key/d70ab142e20f0ba48f80015c273ccc14/";s:11:"script_name";s:31:"/Cybercom/php/magento/index.php";s:4:"skin";s:7:"default";}